### PR TITLE
Step 6: Creating My About Me

### PR DESCRIPTION
**Pull request by leehaney254**
## Step 6: Creating My About Me
In this step I:
- Added aboutme.html file
- Added aboutme.css file
- Updated readme.md

### CODE REVIEW

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
Hi, @leehaney254 ,

Good job so far coming so far with the progress you've made! 💪💪However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great UI design.✔️
- App functions as it should.✔️




# Required Changes ♻️
**Please make sure you condense your project into one HTML file and one CSS file as per the project requirements.**  **So your about.html and myworks.html should be part of your index.html 
Don't forget to add links to the external CSS of these two pages in your index.html**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leehaney254** [comment]
Hello!
I made the requested changes and brought everything to my index.txt file. 
Kindly have a look. Thank you. :smiley:

File: index.html [hunk 1]
```diff
@@ -72,5 +72,166 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
...
     </section>
+    <section id="section-bac">
+      <div id="heading-second">
+        <h1 id="heads">My Recent Works</h1>
+      </div>
+      <section id="sec">
+        <div class="card" id="card">
+          <img src="./assets/first-card.png" alt="card image" />
+          <div class="description">
+            <p id="head1">Multi-Post Stories</p>
```
Comment (OyePriscilla) [inline index.html:83]: - [ ] [optional] Please take note that this suppose to be `hx `tag. 😄

File: index.html [hunk 2]
```diff
@@ -72,5 +72,166 @@ <h1 id="primary">Hey There. <br />I'm Lee</h1>
...
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <div class="buttons">
+              <button class="button1 butcss">css</button>
+              <button class="button1 buthtm">html</button>
+              <button class="button1 butboot">bootstrap</button>
+              <button class="button1 butrub">ruby</button>
+            </div>
```
Comment (OyePriscilla) [inline index.html:95]: - [ ] [optional] Kindly ensure that you wrap the projects technologies with `ul` tag not `div`, each technology in a list `li` tag. `ul `tag is semantics but `div `tag has no meaning. 👍 Kindly apply this to all the projects cards. 😄 .

**Review by OyePriscilla — APPROVED** [review]
Approved 🟢
## Hi @leehaney254  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Highlights ✔️
✅ No Linters error
✅ Descriptive pull request
✅ Great working on the set up the project
✅ Good commit messages
✅ Professional README file

133696182-6c5b784b-0083-427a-b24d-3ec870915f79


### Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @oyepriscilla in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**